ORST/Assets/Scripts/Core/Interactions/ForbiddenSpace/HandHeadProximity.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ORST.Core.Interactions
+{
+    public class HandHeadProximity : MonoBehaviour {
+        public event Action ForbiddenSpaceEntered;
+        public event Action ForbiddenSpaceExited;
+        public event Action<float> ForbiddenSpaceUpdated;
+
+        [SerializeField] private Collider m_InnerCollider;
+        [SerializeField] private Collider m_OuterCollider;
+
+        private readonly List<Transform> m_Intersector = new ();
+        private GameObject m_ForbiddenGameObject;
+
+        private void Update() {
+            ForbiddenSpaceUpdated?.Invoke(GetClosestIntersectLinear());
+        }
+
+        private float GetClosestIntersectLinear() {
+            if (m_Intersector.Count <= 0) {
+                return 0;
+            }
+
+            Vector3 innerColliderPosition =  m_InnerCollider.transform.position;
+            float shortestDist = float.PositiveInfinity;
+            Transform shortestTransform = null;
+            foreach (Transform intersect in m_Intersector) {
+                //
+                if (!m_OuterCollider.bounds.Contains(intersect.position)) {
+                    continue;
+                }
+
+                Vector3 innerColliderDirVec = m_InnerCollider.transform.position - intersect.position;
+                if (!m_InnerCollider.Raycast(new Ray(intersect.position, innerColliderDirVec),
+                                             out RaycastHit hit, innerColliderDirVec.magnitude * 2.0f)) {
+                    //@Maurice - In this case the raycast is unsuccessful because the "intersector" is inside
+                    //the inner collider.
+                    if (m_InnerCollider.bounds.Contains(intersect.position) && m_ForbiddenGameObject == null) {
+                        m_ForbiddenGameObject = intersect.gameObject;
+                        ForbiddenSpaceEntered?.Invoke();
+                    }
+
+                    return 1;
```

Comment:
Same as previous comment
```suggestion
                    return 1.0f;
```